style.css

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
You have not add any stylings here. Please add backgrounds, paddings, uppercase style, font-style, color
![image](https://user-images.githubusercontent.com/104353194/168749703-5f2429aa-89db-4f68-8c3a-0f08f12ec841.png)